app/controllers/adopters_controller.rb

```diff
@@ -99,17 +99,21 @@ def create
   end
 
   def update
+    @title = @adopter.name
+    @adoption_app = @adopter.adoption_app
+    @adopter_users = User.where(edit_my_adopters: true).order("name")
```

Comment:
this seems like it could return a ton of data, not sure the use case though, could it be hundreds?  thousands?

Reply:
Probably, I needed to set these instance vars in case we render the `show` template for the two cases below this. This is exactly what is set in the `show` action. Perhaps pagination could be another thing we can add?

Reply:
This is the existing functionality we have for `show` and it's running pretty smoothly in production.